database/rrddim.c

```diff
@@ -494,7 +497,9 @@ void rrddim_free_custom(RRDSET *st, RRDDIM *rd, int db_rotated)
 
     // free(rd->annotations);
 
+#ifdef ENABLE_ACLK
```

Comment:
Maybe we can use `rrd_memory_mode` instead of doing `#ifdef` in line 503 instead of using `rd->rrd_memory_mode`